mainchain/tx_pool/tx_pool_helper.go

```diff
@@ -0,0 +1,39 @@
+package tx_pool
+
+import (
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/kardiachain/go-kardia/lib/common"
+	"github.com/kardiachain/go-kardia/lib/log"
+)
+
+var UpdateBlacklistInterval uint64 = 50 // blocks since last update
+
+// UpdateBlacklist fetch and overwrite the current blacklist
+func UpdateBlacklist() error {
+	resp, err := http.Get("https://raw.githubusercontent.com/kardiachain/consensus/main/notes")
```

Comment:
Set this Filepath as a const

Reply:
This should have timeout in case network or file slow and server take too much time to do http.get - slow in validate block.
Look into  http.DefaultClient.Timeout or similar setting

Reply:
I have set 1 second timeout for this HTTP request